packages/snap-examples/scripts/buildExamples.js

```diff
@@ -15,48 +12,55 @@ buildExamples();
 async function buildExamples() {
   const examplesDir = await fs.readdir(EXAMPLES_PATH);
 
-  examplesDir.forEach(async (exampleFile) => {
-    const exampleFilePath = resolve(EXAMPLES_PATH, exampleFile);
-    const exampleFileStat = await fs.stat(exampleFilePath);
-
-    if (exampleFileStat.isDirectory()) {
-      const srcPath = resolve(exampleFilePath, 'index.js');
-      const pkgPath = resolve(exampleFilePath, 'package.json');
-      const pkgStat = await fs.stat(pkgPath);
-      const srcStat = await fs.stat(srcPath);
-
-      if (pkgStat.isFile() && srcStat.isFile()) {
-        try {
-          // install dependencies
-          await execa('yarn', ['install'], {
-            cwd: exampleFilePath,
-          });
-        } catch (depsInstallError) {
-          console.error(
-            `Unexpected error when installing dependencies in "${exampleFilePath}.`,
-          );
-          throw depsInstallError;
-        }
+  await Promise.all(
+    examplesDir.map(async (exampleFile) => {
+      const exampleFilePath = pathResolve(EXAMPLES_PATH, exampleFile);
+      const exampleFileStat = await fs.stat(exampleFilePath);
+
+      if (exampleFileStat.isDirectory()) {
+        const srcPath = pathResolve(exampleFilePath, 'src/index.js');
+        const pkgPath = pathResolve(exampleFilePath, 'package.json');
+        const pkgStat = await fs.stat(pkgPath);
+        const srcStat = await fs.stat(srcPath);
 
-        try {
-          await execa(
-            SNAPS_CLI_BIN_PATH,
-            ['build', '--sourceMaps', '--stripComments'],
-            {
+        if (pkgStat.isFile() && srcStat.isFile()) {
+          try {
+            // install dependencies
+            await execa('yarn', ['install'], {
               cwd: exampleFilePath,
-            },
-          );
-        } catch (bundleError) {
-          console.error(
-            `Unexpected error while creating bundle in "${exampleFilePath}.`,
+            });
+          } catch (depsInstallError) {
+            console.error(
+              `Unexpected error when installing dependencies in "${exampleFilePath}.`,
+            );
+            throw depsInstallError;
+          }
+
+          try {
+            await execa(
+              'node',
+              [
+                pathResolve(exampleFilePath, SNAPS_CLI_BIN_PATH),
```

Comment:
Here we use `node` instead of invoking the file directly because it may node be directly executable by the time this script runs. Invoking it with `node` always works though.